### DISCUSSION TITLE
Traefik's Default Cert is "Not Secure"

### DISCUSSION THREAD

**Discussion by bonzo72** [techno-tim/techno-tim.github.io#141] [Questions]
Not sure how to fix this. Can't seem to get the browser (or any browser) to load the dashboard in https. Probably something dumb I'm missing. It was saying that the cert was "invalid". So I tried, per the Certificate instructions, to install it in the Trusted Root CA group. But that resulted in the image below: 

![traefik](https://user-images.githubusercontent.com/105714515/168791588-745a93a4-14e3-4b58-83d1-af272f96ed82.png)

**Comment by contactnat** [comment]
a couple of us had this issue not too long ago and we stumbled across this: https://community.traefik.io/t/problem-renewing-the-cert-with-dns-01-challenge/8305 which led us to the change.  
adding the following line to the traefik.yml fixes it: disablePropagationCheck: true
By setting this flag to true disables the need to wait for the propagation of the TXT record to all authoritative name servers. 

e.g.:
certificatesResolvers:
  cloudflare:
    acme:
      email: myemail@gmail.com
      storage: acme.json
      dnsChallenge:
        provider: cloudflare
        disablePropagationCheck: true
        resolvers:
          - "1.1.1.1:53"
          - "8.8.8.8:53"
